### PR TITLE
WIP BXMSDOC-6500: Modified sentence related to @watch annotation usage

### PR DESCRIPTION
**Pull request by hmanwani-rh**
See JIRA: https://issues.redhat.com/browse/BXMSDOC-6500

Doc preview:
[Designing a decision service using spreadsheet decision tables RHPAM 7.9](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6500-RHPAM/#decision-tables-rule-table-entries-ref)
[Designing a decision service using spreadsheet decision tables RHDM 7.9](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-6500-RHDM/#decision-tables-rule-table-entries-ref)

### CODE REVIEW

**Review by mariofusco — APPROVED** [review]

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-rule-table-entries-ref.adoc [hunk 1]
```diff
@@ -50,7 +50,8 @@ To include a pattern without constraints, you can write the pattern in front of
 +
 If the pattern ends with `eval`, code snippets produce boolean expressions for inclusion into a pair of parentheses after `eval`.
 +
-The pattern can terminate with the `@watch` annotation, used to customize the properties on which the pattern is reactive.
+You can terminate the pattern with `@watch` annotation, which is used to customize the properties on which the pattern is reactive.
```
Comment (michelehaglund) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-rule-table-entries-ref.adoc]: How about this minor rewrite that removes on instance of "which"?
You can terminate the pattern with the @watch annotation, which is used to customize the properties that the pattern is reactive on.

**Review by michelehaglund — APPROVED** [review]
Looks good @hmanwani-rh Minor suggestion in line.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-rule-table-entries-ref.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-rule-table-entries-ref.adoc]: Done, thanks Michele

**Review by hellowdan — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**